### PR TITLE
rtmp-services: Add VIMM

### PR DESCRIPTION
**Pull request by chirenonsteem**
VIMM is a decentralized video platform for gamers and independent creators. Visit VIMM.TV to learn more.

### Description
Added VIMM to services list with new ingest URLs

### Motivation and Context
This makes getting started on VIMM easier for streamers by optimizing some settings and allowing them to select their region for better performance.

### How Has This Been Tested?
Tested on WIN10 on 2020-05-20. Tested each ingest server by doing a test stream.

### Types of changes

- New feature (non-breaking change which adds functionality)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -216,6 +216,26 @@
                 "max audio bitrate": 160
             }
         },
+        {
+            "name": "VIMM",
+            "common": true,
```
Comment (notr1ch) [inline plugins/rtmp-services/data/services.json:221]: This is not a common service.

File: plugins/rtmp-services/data/services.json [hunk 2]
```diff
@@ -216,6 +216,26 @@
...
+            "servers": [
+                {
+                    "name": "Europe: Frankfurt",
+                    "url": "rtmp://us.vimm.tv/live"
```
Comment (notr1ch) [inline plugins/rtmp-services/data/services.json:225]: This is the same URL as "North America: Montreal".

File: plugins/rtmp-services/data/services.json [hunk 3]
```diff
@@ -216,6 +216,26 @@
                 "max audio bitrate": 160
             }
         },
+        {
+            "name": "VIMM",
+            "common": false,
```
Comment (notr1ch) [inline plugins/rtmp-services/data/services.json:221]: The line can be removed entirely, since the default is false.

Comment (chirenonsteem) [reply plugins/rtmp-services/data/services.json:221]: Will do, thank you.

**Comment by chirenonsteem** [comment]
@notr1ch I made the changes and squashed the commits. But I'm still wondering what branch I should use? Should I just create one?

**Comment by notr1ch** [comment]
It's fine for now on master, but in future you should always prefer to start a PR on a separate branch. This way you can make updates to the master branch without affecting the PR, and a separate branch allows maintainers to push fixes to the PR branch before merging.

**Comment by jp9000** [comment]
Manually fixing and merging.  Please don't use master branch of the fork next time.  You'll see "closed in [commit]".

**Pull request closed without merging**